### DISCUSSION TITLE
Source of Land cover data in example data

### DISCUSSION THREAD

**Discussion by rubcalvo** [JGCRI/xanthos#73] [Q&A]
Hi to all! I could not find the source of the land cover data and the GCAM parameters for the penman-monteith estimation that comes with the example data from https://zenodo.org/records/2578287. Does anyone know where I can find this information?

Any info would be greatly appreciated!

**Comment by mgaacGitHub** [comment]
Dear rubcalvo, i would like to know if could you find the source of the land cover data and if do you know to what type of land cover/use corresponde the eight land classes. Greetings ! Miguel
